examples/material_gallery/lib/gallery/home.dart

```diff
@@ -104,18 +104,18 @@ class GalleryHomeState extends State<GalleryHome> {
         children: <Widget>[
           new TwoLevelSublist(
             leading: new Icon(icon: Icons.star),
-            title: new Text("Demos"),
+            title: new Text('Demos'),
             children: <Widget>[
-              new GalleryItem(title: "Weather", builder: () => new WeatherDemo()),
-              new GalleryItem(title: "Fitness", builder: () => new FitnessDemo()),
-              new GalleryItem(title: "Fancy lines", builder: () => new DrawingDemo()),
+              new GalleryItem(title: 'Weather', builder: () => new WeatherDemo()),
+              new GalleryItem(title: 'Fitness', builder: () => new FitnessDemo()),
+              new GalleryItem(title: 'Fancy lines', builder: () => new DrawingDemo()),
               new GalleryItem(title: 'Flexible space toolbar', builder: () => new FlexibleSpaceDemo()),
               new GalleryItem(title: 'Floating action button', builder: () => new TabsFabDemo()),
```

Comment:
I think this list should be ordered mightiest-first. I would guess that most people will only try a few demos and they're most likely to try the first couple from this list. We may want to reconsider the order and  even the contents of this list.